ruby/facter

```diff
@@ -1 +1 @@
-Subproject commit 8ff4fa5e0251aaa1b3fea1e2e308a34b2ffe6027
+Subproject commit 691e2b164ec4144d0ce80c14d3132e89242f26cc
```

Comment:
Looks like this would bump up our Puppet Server 2.0 facter submodule to 2.4.3.  @kylog - does this sound like a reasonable version of Facter to be testing against for our latest Puppet Server 2 / AIO related work?